dagmc_stats.py

```diff
@@ -201,3 +203,43 @@ def get_triangle_aspect_ratio(my_core, meshset, geom_dim):
         t_a_r.append(top/bottom)
 
     return t_a_r
+
+def get_area_triangle(my_core, meshset):
```

Comment:
This function has many similarities to the function above `get_triangle_aspect_ratio`. The only difference seems to be the formula used to calculate area or aspect ratio. The two could probably be combined into a single function (such as `get_triangle_stats` or similar) and then only calls functions that do the two different maths like `calc_aspect_ratio` and `calc_tri_area`.

```
def calc_aspect_ratio(...):
    tar = ...
    return tar

def calc_tri_area(...):
    area = ...
    return area

def get_triangle_stats(...):
     # do all the work one time to collect the list of verts and calculating a, b, c for the triangle
     if TAR:
         calc_aspect_ratio()
     if area:
         calc_tri_area()
    return ...
```
That's one example to consolidate these two functions. There may be a better/different way. But the ultimate goal should be to reduce redundant code.